config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -455,7 +455,7 @@ presubmits:
           exit $result
         securityContext:
           privileged: true
-  - name: pull-cluster-api-provider-azure-e2e-workload-upgrade-1-22-latest-main
+  - name: pull-cluster-api-provider-azure-e2e-workload-upgrade
```

Comment:
So we're turning this into a "mutable" upgrade test whose specifics will be maintained in the capz repo itself? (instead of specifying "from" and "to" versions in this yaml spec?)

Reply:
correct (using the default upgrade versions defined in azure-dev.yaml)